src/commands/preboot-paired-simulator.yml

```diff
@@ -4,18 +4,31 @@ parameters:
     description: The device type. E.g., "iPhone 13 Pro Max"
     type: string
     default: "iPhone 13 Pro Max"
+  iphone-device-udid-var:
+    description: The name of the variable to contain the iphone's UDID e.g. IPHONE_UDID
+    type: string
+    default: IPHONE_UDID
```

Comment:
Same as my comment in the other file, let's make these something like `MACOS_ORB_<type>_UDID` by default

Reply:
done